src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```
@@ -0,0 +1,111 @@
+                String country = resultSet.getString("country");
+                manufacturer = new Manufacturer(idOfManufacturer, name, country);
+            }
+
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() throws RuntimeException {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement
+                    .executeQuery(
+                        "SELECT manufacturers.id, manufacturers.name, manufacturers.country"
+                        + "  FROM manufacturers WHERE is_deleted = 0");
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer = new Manufacturer(id, name, country);
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public void delete(Long id) {
+        String deleteRequest = "UPDATE manufacturers SET is_deleted = 1 where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteStatement =
+                         connection.prepareStatement(
+                                 deleteRequest, Statement.RETURN_GENERATED_KEYS)) {
+            deleteStatement.setLong(1, id);
+            deleteStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
```

Exception message does not contain error information.
throw new RuntimeException("Error for delete method", e);
But accordind task:
"Let's create custom exception DataProcessingException and constructor with two parameters: String message and Throwable ex.
It should be extended from RuntimeException. You should rethrow this exception in catch block on dao layer."